docs/android_studio.md

```diff
@@ -1,38 +1,3 @@
 ---
-title: Running from Android Studio
+redirect_to: http://flutter.dev/docs/development/tools/devtools/android-studio
```

Comment:
nit: these could be https to save an extra redirect

Reply:
done